mono/sgen/sgen-gc.c

```diff
@@ -3315,6 +3315,16 @@ init_sgen_mode (SgenMode mode)
 	init_sgen_major (major);
 }
 
+void
+initialize_sgen_gc_info() 
+{
+	sgen_gc_info.fragmented_bytes = 0;
+	sgen_gc_info.heap_size_bytes = 0;
+	sgen_gc_info.high_memory_load_threshold_bytes = 0;
+	sgen_gc_info.memory_load_bytes = 0;
+	sgen_gc_info.total_available_memory_bytes = 0;
```

Comment:
Are these just globals, that are already 0? I.e. this function probably not needed.

Reply:
Yeah, you're right.